### PR TITLE
redesigned warning alert

### PR DESCRIPTION
**Pull request by ShreshthaKriti**
FIN-21: https://github.com/FuturICT2/FIN4Xplorer/issues/21

### CODE REVIEW

**Comment by benjaminaaron** [comment]
I get `Module not found: Can't resolve '@material-ui/lab/Alert'`. Did you forget to commit your changes in `package.json`?

File: src/components/TopBar.jsx [hunk 1]
```diff
@@ -48,6 +49,12 @@ const useStyles = makeStyles(theme => ({
 		width: '20px',
 		height: '20px',
 		padding: '0 4px 4px 0'
+	},
+	alert: {
+		width: '50%',
+		'& > * + *': {
```
Comment (benjaminaaron) [inline src/components/TopBar.jsx]: Oha, what is this?

**Review by benjaminaaron — CHANGES_REQUESTED** [review]
Looks nice, cool choice with the `Alert` package 👍

Can you make the content of the alert box centered? This is how it looks for me and there is empty space on the right side:

Screenshot 2020-02-19 at 17 43 06

Also, as written on the Trello card, it would be great if the box is not in the `TopBar` but below it. That requires moving it out of `TopBar.jsx` and somewhere else where it can always be displayed not matter on which subsite the user is. Probably the right place is in `App.jsx` below `<TopBar />` 🤔

File: src/components/TopBar.jsx (hunk 1 above)
Comment (ShreshthaKriti) [reply src/components/TopBar.jsx]: Ah, that was me experimenting some stuff. I moved the warning to a different component. Regarding the text centering - i did a temporary fix  >.< That will break if you open the dev tools and change the width of the page. 
The previous one was working fine even with varying width of the page. But for some reason the text is shifted to a bit left in normal view.

File: src/components/TopBar.jsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faQrcode } from '@fortawesome/free-solid-svg-icons';
 import { Link } from 'react-router-dom';
 import QRModal from './QRModal';
+import Alert from '@material-ui/lab/Alert';
```
Comment (benjaminaaron) [inline src/components/TopBar.jsx]: Obsolete import

File: src/components/Warning.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Alert from '@material-ui/lab/Alert';
+import colors from '../config/colors-config';
```
Comment (benjaminaaron) [inline src/components/Warning.jsx]: Unused import

File: src/components/TopBar.jsx [hunk 4]
```diff
@@ -48,6 +49,14 @@ const useStyles = makeStyles(theme => ({
...
 		height: '20px',
 		padding: '0 4px 4px 0'
+	},
+	alert: {
+		width: '50%'
+	},
+	message: {
+		// marginLeft: '40px',
+		margin: 'auto',
+		textAlign: 'center'
```
Comment (benjaminaaron) [inline src/components/TopBar.jsx]: Since you outsourced the alert to its own component we don't need these additions anymore in `TopBar`.

**Review by benjaminaaron — CHANGES_REQUESTED** [review]
🔹 The addition to `package.json` is still missing
🔹 Please center the content of the alert box, there is too much empty space on the right side of it
🔹 You forgot to bring along the condition for showing the alert box or not (`props.defaultAccount === null`). It is now always shown, even when MetaMask is connected to the right network.

**Pull request closed without merging**